content/docs/other-guides/support.md

```diff
@@ -0,0 +1,168 @@
+          the size of the community for that application and the current level
+          of active development of the application.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Support from a cloud provider
+
+If you're using the services of a cloud provider to host Kubeflow, the cloud
+provider may be able to help you diagnose and solve a problem.
+
+Consult the support page for the cloud service that you're using:
+
+* [Amazon Web Services (AWS)](https://aws.amazon.com/contact-us/)
+* [Google Cloud Platform (GCP)](https://cloud.google.com/support-hub/)
+* [IBM Cloud Private](https://www.ibm.com/cloud/support)
+* [Microsoft Azure](https://azure.microsoft.com/en-au/support/options/)
+
+## Support from the Kubeflow community
+
+Kubeflow has an active and helpful community of users and contributors. You
+can ask questions and make suggestions in the following places:
+
+* **Slack** for online chat and messaging. See details of Kubeflow's 
+  [Slack workspace and channels](/docs/about/community/#slack).
+
+* **Kubeflow discuss** for email-based group discussion. Join the
+  [kubeflow-discuss](https://groups.google.com/forum/#!forum/kubeflow-discuss) 
+  group.
+
+* **Kubeflow documentation** for overviews and how-to guides. In particular,
+  refer to the following documents when troubleshooting a problem:
+
+  * [Kubeflow installation and setup](/docs/started/getting-started/)
+  * [Kubeflow components](/docs/components/)
+  * [Further setup and troubleshooting](/docs/other-guides/)
+
+* **Kubeflow issue trackers** for known issues, questions, and feature requests.
+  Search the open issues to see if someone else has already logged the problem 
+  that you're encountering and learn about any workarounds to date. If no-one
+  has logged your problem, create a new issue to describe the problem.
+
+    Each Kubeflow application has its own issue tracker within the [Kubeflow
+    organization on GitHub](https://github.com/kubeflow). To get you started,
+    here are the primary issue trackers:
+
+  * [Kubeflow core](https://github.com/kubeflow/kubeflow/issues)
+  * [kfctl command-line tool](https://github.com/kubeflow/kfctl/issues)
+  * [Kustomize manifests](https://github.com/kubeflow/manifests/issues)
+  * [Kubeflow Pipelines](https://github.com/kubeflow/pipelines/issues)
+  * [Katib hyperparameter tuning](https://github.com/kubeflow/katib/issues)
+  * [Metadata](https://github.com/kubeflow/metadata/issues)
+  * [Fairing notebook SDK](https://github.com/kubeflow/fairing/issues)
+  * [TensorFlow training (TFJob)](https://github.com/kubeflow/tf-operator/issues)
+  * [PyTorch training (PyTorchJob)](https://github.com/kubeflow/pytorch-operator/issues)
+  * [KFServing](https://github.com/kubeflow/kfserving/issues)
+  * [Examples](https://github.com/kubeflow/examples/issues)
+  * [Documentation](https://github.com/kubeflow/website/issues)
+
+## Support from partners
+
+TODO Shall we include a partners section like this?
```

Comment:
Noting the TODO.

@jbottum and @abhi-g What are your thoughts about including a "partners" section?

Reply:
I'd like to have this section. Though we may want to consider the name "Partners". That generally implies a business relationship and none of those providers will have a business relation with Kubeflow. Couple of options: "Ecosystem Providers", "Kubeflow Collaborators".
/cc @theadactyl

Reply:
Thanks @abhi-g that's a good point. I've changed the wording from "Support from partners" to "Support from providers in the Kubeflow ecosystem".